cmake/WlFunctions.cmake

```
@@ -104,7 +104,7 @@ macro(_common_compile_tasks)
 
   if(ARG_USES_MINIZIP)
       if(MINIZIP_STATIC_LIBRARIES)
-          target_link_libraries(${NAME} minizip)
+          target_link_libraries(${NAME} ${pkgcfg_lib_MINIZIP_minizip})
           message(STATUS "Link ${NAME} with minizip")
```

I would prefer to use `MINIZIP_LINK_LIBRARIES` as it is a documented variable, I could not find anything about `pkgcfg_lib_*` in cmake documentation (https://cmake.org/cmake/help/latest/module/FindPkgConfig.html#command:pkg_check_modules). 
```suggestion
          target_link_libraries(${NAME} ${MINIZIP_LINK_LIBRARIES})
          message(STATUS "Link ${NAME} with ${MINIZIP_LINK_LIBRARIES}")
```
Does this still work for you?

---

> I would prefer to use `MINIZIP_LINK_LIBRARIES` as it is a documented variable, I could not find anything about `pkgcfg_lib_*` in cmake documentation (https://cmake.org/cmake/help/latest/module/FindPkgConfig.html#command:pkg_check_modules).
> 
> Does this still work for you?

I would have used that if I had seen it in CMakeCache.txt. The contents of the variable are closer to what I was looking for.